packages/api/src/command/medical/tcm-encounter/create-sample-tcm-encounter.ts

```diff
@@ -0,0 +1,49 @@
+import { buildDayjs } from "@metriport/shared/common/date";
+import { sample } from "lodash";
+import { createTcmEncounter } from "./create-tcm-encounter";
+
+export async function createSampleTcmEncounters(cxId: string, patientId: string) {
```

Comment:
This is very unusual in the codebase, so could we:
- add a TSDoc explaining it's just for non-production flows, that it actually adds sample data to the cx's DB; and
- rename to indicate it stores those in the DB/storage?

At least the TSDoc, not sure about the second one. Leaving it to you.